scss/talis.scss

```diff
@@ -0,0 +1 @@
+@import "custom";
```

Comment:
Here, we're restoring a CSS file that has exactly the same contents as the `custom.scss` file that lives alongside it.

We cannot be sure that `talis.css` will remain identical after this change, but at least it is recreated from source.